psdm_qs_cli/QuestionnaireClient.py

```diff
@@ -337,8 +337,7 @@ def lookupByExperimentName(self, experiment_name):
         Given an experiment name, try to get the best guess as to the proposal_id and run period.
         """
         r = self.rget(
-            self.questionnaire_url + "ws/questionnaire/lookupByExperimentName",
-            {"experiment_name": experiment_name},
+            "https://pswww.slac.stanford.edu/ws-kerb/lgbk/lgbk/ws/experiments_to_proposal"
```

Comment:
We need to do a bit more here if we want to have this function do what it was meant to.  Currently this endpoint gives us a json mapping:
```
{ experiment_name : 
    {
        'name' : experiment_name,
        'instrument' : instrument,
        'proposal_id' : proposal_id
    }
}
````
While we're here, could you add some code to this function to grab the proposal ID given the experiment name?